### PR TITLE
Warning banner for Cat C and D +Es

### PR DESCRIPTION
**Pull request by amobarak**
## Description

Fixed issue with always using C or D category in vehicle checks modal also showing warning banner in modal for C and D +e categories

## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by rossfauldsbjss — APPROVED** [review]
looks good

File: src/pages/waiting-room-to-car/cat-c/components/vehicle-checks-modal/vehicle-checks-modal.cat-c.page.ts [hunk 1]
```diff
@@ -185,10 +185,23 @@ export class VehicleChecksCatCModal {
     this.store$.dispatch(new TellMeQuestionOutcomeChanged(result, index));
   }
 
-  shouldDisplayBanner = (): boolean => {
+  nonTrailerBanner() {
     return (
       this.vehicleChecksScore.drivingFaults === 4 &&
-      this.vehicleChecksScore.seriousFaults === 1
+       this.vehicleChecksScore.seriousFaults === 1 &&
```
Comment (szabikr) [inline src/pages/waiting-room-to-car/cat-c/components/vehicle-checks-modal/vehicle-checks-modal.cat-c.page.ts]: indentation might cause the build to fail

Comment (szabikr) [reply src/pages/waiting-room-to-car/cat-c/components/vehicle-checks-modal/vehicle-checks-modal.cat-c.page.ts]: also the name of the function should be more specific about the action that it does. And please add a return type to it

File: src/pages/waiting-room-to-car/cat-d/components/vehicle-checks-modal/vehicle-checks-modal.cat-d.page.ts [hunk 2]
```diff
@@ -222,10 +222,23 @@ export class VehicleChecksCatDModal {
     this.store$.dispatch(new SafetyQuestionOutcomeChanged(result, index));
   }
 
-  shouldDisplayBanner = (): boolean => {
+  nonTrailerBanner() {
```
Comment (szabikr) [inline src/pages/waiting-room-to-car/cat-d/components/vehicle-checks-modal/vehicle-checks-modal.cat-d.page.ts]: Same here

**Review by szabikr — APPROVED** [review]

**Pull request merged**